recipes/nextclade/meta.yaml

```diff
@@ -14,9 +14,11 @@ source:
     sha256: 6f8e5ad0503c3f30d433747ee19727c55db614fc04cf708887921bb2f5053ebb                              # [arm64]
 
 build:
-  number: 0
+  number: 1
 
 requirements:
+  build:
+    - curl
```

Comment:
I think this needs to be host

Reply:
This is only used at build time so should it not be [build](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#build)? I thought host things were things that "need to be specific to the target platform when the target platform is not necessarily the same as the native build platform" (see [here](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#host))